SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2104,8 +2104,7 @@ void USpatialReceiver::ProcessQueuedActorRPCsOnEntityCreation(AActor* Actor, RPC
 	{
 		UFunction* Function = Info.RPCs[RPC.Index];
 		const FRPCInfo& RPCInfo = ClassInfoManager->GetRPCInfo(Actor, Function);
```

Comment:
This is looking up the RPC in the wrong ClassInfo. We should pass the entity ID into this function and for each RPC get the target object and then look up the RPC info.